test/WebSites/ApiExplorerWebSite/ApiExplorerDataFilter.cs

```diff
@@ -9,11 +10,11 @@
 namespace ApiExplorerWebSite
 {
     /// <summary>
-    /// An action filter that looks up and serializes Api Explorer data for the action.
+    /// A resource filter that looks up and serializes Api Explorer data for the action.
     ///
     /// This replaces the 'actual' output of the action.
     /// </summary>
-    public class ApiExplorerDataFilter : ActionFilterAttribute
+    public class ApiExplorerDataFilter : IResourceFilter
```

Comment:
Wondering if there is some other reason for this other than  we want this to run before model binding?? (so that we can just consume it without actually invoking the action?)